ClosedXML/Excel/XLWorksheet.cs

```diff
@@ -1491,5 +1492,19 @@ public Object Evaluate(String expression)
         }
 
         public String Author { get; set; }
+
+        public List<Drawings.IXLPicture> Pictures()
+        {
+            return pictures;
+        }
+
+        public void AddPicture(Drawings.XLPicture pic)

```

Comment:
🔴 I am thinking if List is a good choice here ? what if i add a same picture more than once, would be nice if we can check for item you are trying to add and add only if it doesnot exisits ,, we have enogh memory issues at a moment